chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,922 @@
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"CHP-009321",
+										"measures"
+									]
+								},
+								"description": "GET /{chplPrefix}-{identifier}/measures end point should return status 400 for requests without API Key"
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "GET: /{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode}",
+					"item": [
+						{
+							"name": "GET /certified_products/{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode} -valid API Key returns all RWT fields",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/certified_products/{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode} end point should return status 200 for valid API Key\", function () {\r",
+											"    pm.response.to.have.status(200);\r",
+											"});\r",
+											"const expresp = pm.response.json()\r",
+											"pm.test(\"/certified_products/{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode} end point response should have all RWT fields\", function () {\r",
+											"    pm.expect(expresp).to.have.property('rwtPlansUrl');\r",
+											"    pm.expect(expresp).to.have.property('rwtPlansCheckDate');\r",
+											"    pm.expect(expresp).to.have.property('rwtResultsUrl');\r",
+											"    pm.expect(expresp).to.have.property('rwtResultsCheckDate');\r",
+											"    pm.expect(expresp).to.have.property('rwtEligibilityYear');\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"value": "{{apiKey}}",
+										"type": "text"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/15.02.02.2270.A106.02.01.0.210112",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"15.02.02.2270.A106.02.01.0.210112"
+									]
+								},
+								"description": "GET /certified_products/{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode} end point returns status 200 and response should have all RWT fields for valid API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /certified_products/{year}.{testingLab}.{certBody}.{vendorCode}.{productCode}.{versionCode}.{icsCode}.{addlSoftwareCode}.{certDateCode} - Invalid API Key",
```

Comment:
There seems to be a lot of duplication of these tests. Is there any way to simplify these, probably using the data-driven idea?

Reply:
Removed duplicated tests and added data driven tests for all end points